R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R

```diff
@@ -227,9 +220,9 @@ module_aglu_LA107.an_IMAGE_R_C_Sys_Fd_Y <- function(command, ...) {
                      "temp-data-inject/L100.IMAGE_an_Prodmixfrac_ctry_C_Y",
                      "temp-data-inject/L100.IMAGE_an_Feedfrac_ctry_C_Sys_Fd_Y",
                      "temp-data-inject/L100.IMAGE_an_FeedIO_ctry_C_Sys_Y",
-                     "temp-data-inject/L105.an_Prod_Mt_ctry_C_Y") %>%
+                     "L105.an_Prod_Mt_ctry_C_Y") %>%
       # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_PROTECT_FLOAT) ->
```

Comment:
Using output instead of the temp inject data needs `protect_float` to pass test.